File: Problem Solving/CodeForces/ICPC Assiut University Training Sheet/Newcomers/Java/C-Sheet #2-Loop/I.java
import java.util.Scanner;

public class I {
  public static void main(String[] args){
  Scanner input = new Scanner(System.in);
  int x,y;
  int z = 0;
  x = input.nextInt();
  y = x;
  while (x>0){
    z = z * 10 + x % 10;
    x = x / 10;
  }
  if (y == z){
    System.out.print(z + "\n" + "YES");
  } else {
    System.out.print(z + "\n" + "NO" );
  } 
  input.close();
  }
}

File: Problem Solving/CodeForces/ICPC Assiut University Training Sheet/Newcomers/C/C-Sheet #2-Loop/I.c
#include <stdio.h>

int main() {
  int x,y;
  int z = 0;
  scanf("%d",&x);
  y = x;
  while (x){
    z = z * 10 + x % 10;
    x = x / 10;
  }
  if (y == z){
  printf("%d\n YES\n",z);
  } else {
    printf("%d\n NO\n",z);
  } 

  return 0;
}
